src/test/java/org/opentripplanner/raptor/_data/api/TestRaptorPath.java

```diff
@@ -11,6 +11,11 @@
 import org.opentripplanner.raptor.api.path.RaptorStopNameResolver;
 import org.opentripplanner.raptor.api.path.TransitPathLeg;
 
+/**
+ * This can be used to test the comparators and ParetoSet. Please use the real
+ * {@link org.opentripplanner.raptor.path.Path} implementation is other
```

Comment:
(GitHub doesn't like to play ball with me on this commit.)

`{@link org.opentripplanner.raptor.path.Path} implementation is other`
->
`{@link org.opentripplanner.raptor.path.Path} implementation if other`

Reply:
I have fixed it